js/deer-config.js

```diff
@@ -183,6 +183,7 @@ export default {
                             li.setAttribute("deer-id", glossID)
                             let a = document.createElement("a")
                             a.setAttribute("href", options.link+glossID)
+                            a.setAttribute("target", "_blank")
```

Comment:
We'll have to watch this so it doesn't create tab litter under heavy use, but I get why it is there